test/Codenizer.HttpClient.Testable.Tests.Unit/WhenMatchingRoutes.cs

```diff
@@ -9,14 +13,18 @@ namespace Codenizer.HttpClient.Testable.Tests.Unit
     public static class ConfiguredRequestsExtensions
     {
         internal static RequestBuilder? Match(this ConfiguredRequests configuredRequests, HttpMethod method, string uri,
-            string? accept)
+            string? accept, AuthenticationHeaderValue? auth = null)
```

Comment:
I'm not sure of the blast radius but I'd like to avoid the default parameter value here.
Could you take a look at omitting that and passing `null` in the relevant call sites?